extra/resources/controld.in

```diff
@@ -260,17 +263,19 @@ controld_monitor() {
 }
 
 controld_validate() {
-    check_binary killall
-    check_binary "${OCF_RESKEY_daemon}"
-
     case "${OCF_RESKEY_CRM_meta_globally_unique}" in
         yes|Yes|true|True|1)
             ocf_log err "$OCF_RESOURCE_INSTANCE must be configured with the globally_unique=false meta attribute"
             exit $OCF_ERR_CONFIGURED
             ;;
     esac
 
-    [ -d "@runstatedir@/cluster" ] || mkdir "@runstatedir@/cluster"
+    # Host-specific checks
+    if [ "$OCF_CHECK_LEVEL" = "10" ]; then
+        check_binary killall
+        check_binary "${OCF_RESKEY_daemon}"
+        [ -d "@runstatedir@/cluster" ] || mkdir "@runstatedir@/cluster"
```

Comment:
Ouch, I just realized this was a little hacky before -- not only did it validate, but it created this directory if needed. start/monitor call validate (without a check level). So we still need to do this, but it doesn't belong in the validate function

Reply:
Ahhh yep.